src/modules/home/components/header/user-navigation.menu.tsx

```diff
@@ -21,6 +26,8 @@ interface MenuProps {
 export function UserNavigationMenu<T extends MenuProps>({ anchorEl, handleMenuClose, handleMenuOpen }: T) {
   const [isLanguageModalOpen, setIsLanguageModalOpen] = useState(false);
   const [selectedLanguage, setSelectedLanguage] = useState<Language>(Language.UZ);
+  const auth = useAppSelector((state) => state.auth);
```

Comment:
I think it's a beneficial idea to create a selector callback. Because this will be used frequently. Something like:
in auth-slice.ts: `export const selectAuth = (state) => state.auth;`
and use it as: `const auth = useAppSelector(selectAuth);`